lms/templates/verify_student/make_payment_step.underscore

```diff
@@ -30,7 +30,7 @@
     <div class="instruction <% if ( !upgrade && isActive ) { %>center-col<% } %>">
       <% if ( _.some( requirements, function( isVisible ) { return isVisible; } ) ) { %>
       <p class="instruction-info">
-        <% if ( verificationDeadline ) { %>
+        <% if ( verificationDeadline && isActive) { %>
```

Comment:
@awaisdar001 I added another test that checks when user is activated and verification deadline is also set. Please have review again. Thanks